tests/wpt/web-platform-tests/resources/testharness.js

```diff
@@ -921,21 +921,23 @@ policies and contribution forms [3].
 
              var p;
              for (p in actual) {
-                 assert(expected.hasOwnProperty(p), "assert_object_equals", description,
+                 assert(p in expected, "assert_object_equals", description,
                                                     "unexpected property ${p}", {p:p});
 
                  if (typeof actual[p] === "object" && actual[p] !== null) {
                      if (stack.indexOf(actual[p]) === -1) {
                          check_equal(actual[p], expected[p], stack);
                      }
+                 } else if (p == "lastModified") {
+                    // don't check creation-time related properties
```

Comment:
When `lastModified` is unknown during `File` construction, the https://w3c.github.io/FileAPI/#dfn-lastModified says that we should use current time (and I implemented so in File API). so it is a property impossible to make equal.